landoscript/src/landoscript/actions/android_l10n_import.py

```diff
@@ -100,6 +108,25 @@ async def run(
 
             diff += diff_contents(orig_file, new_file, l10n_file.dst_name)
 
+        for info in android_l10n_import_info.toml_info:
+            src_path = info.toml_path
+            dst_path = f"{info.dest_path}/l10n.toml"
+
+            if src_path not in new_files:
+                log.warning(f"WEIRD: {src_path} not in new_files...continuing anyways...")
+
+            if dst_path not in orig_files:
+                log.warning(f"WEIRD: {dst_path} not in orig_files...continuing anyways...")
+
+            orig_file = orig_files[dst_path]
+            new_file = new_files[src_path]
```

Comment:
This will crash if we hit one of the warnings above?  Seems like it would be better to either remove the warning or raise right away.